### PR TITLE
MEN-5715: Webhooks integration

### PR DESCRIPTION
**Pull request by alfrunes**
Goes together with:
 * mendersoftware/workflows#231
 * mendersoftware/deviceauth#610

### CODE REVIEW

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline --fast

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline --pr workflows/231 --pr deviceauth/610

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline --pr workflows/231 --pr deviceauth/610

**Comment by alfrunes** [comment]
@mender-test-bot start pipeline --pr workflows/231 --pr workflows-enterprise/425 --pr deviceauth/610 --pr deviceauth-enterprise/100

File: docs/management_api.yml [hunk 1]
```diff
@@ -434,6 +441,23 @@ components:
...
 
+    HTTP:
+      type: object
+      description: |
+        HTTP Webhook configuration.
+      properties:
+        url:
+          type: string
+          description: >-
+            The destination URL the for the webhook.
```
Comment (kjaskiewiczz) [inline docs/management_api.yml:452]:
```suggestion
            The destination URL for the webhook.
```

**Review by kjaskiewiczz — APPROVED** [review]
looks good; great job!

**Pull request merged**